scripts/pullModels.js

```diff
@@ -1,10 +1,15 @@
+const fs = require('fs');
+const path = require('path');
 const download = require('download');
 const { modulesOSSUrl } = require('./config');
- 
+
 (async () => {
   Object.keys(modulesOSSUrl).forEach(item => {
     modulesOSSUrl[item].forEach(itm => {
       download(itm, `dist/static/models/${item}`);
     });
   });
+  fs.writeFile(path.resolve(process.cwd(), './dist/CNAME'), 'pipboard.imgcook.com', (err) => {
```

Comment:
This file is called pullModels, where we shouldn't do write this :)

Reply:
good suggestion